doc/user-guide/password.md

```diff
@@ -174,33 +239,32 @@ You can set the following variables in your `.env` file to customize behavior:
 :class-container: text-center
 :gutter: 2
 
-
 :::{grid-item-card} Panel
-:link: https://github.com/ploomber/doc/tree/main/examples/panel/app-with-auth0
+:link: <https://github.com/ploomber/doc/tree/main/examples/panel/app-with-auth0>
```

Comment:
I think these `<>` broke the links https://ploomber-doc--283.org.readthedocs.build/en/283/user-guide/password.html#examples

be careful when making these changes

Reply:
> I think these `<>` broke the links https://ploomber-doc--283.org.readthedocs.build/en/283/user-guide/password.html#examples
> 
> be careful when making these changes

[removed](https://github.com/ploomber/doc/pull/283/commits/a482d5a1059c5633c7ab54a7947d628ce78a0395#diff-ee5b667893ae9f81da782a6407f629f366501a0e2688f6ba2477c70cd8963a5cR243)